src/app/pages/RewardPage/components/RewardPaginaton/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { translations } from 'locales/i18n';
+import { Button, ButtonStyle } from 'app/components/Button';
+
+interface IRewardPaginationProps {
+  page: number;
+  loading: boolean;
+  isDisabled: boolean;
+  onChange: (value: number) => void;
+}
+
+export const RewardPagination: React.FC<IRewardPaginationProps> = ({
```

Comment:
I'd like to see also `First` button (in case you're on page 2 or more), pls discuss it with Pili and Gilad.

Reply:
added